friends/src/components/FriendList.jsx

```diff
@@ -106,9 +160,17 @@ export default class FriendList extends React.Component {
 				</AddFriendButton>
 				<FormContainerDiv>
 					<FriendForm
+						componentUpdated={this.componentUpdated}
+						shouldUpdate={this.state.shouldUpdate}
 						isEditing={this.state.isEditing}
 						length={this.state.length}
 						addNewFriend={this.addNewFriend}
+						isUpdating={this.state.isUpdating}
+						putFriend={this.putFriend}
+						name={this.state.name}
+						age={this.state.age}
+						email={this.state.email}
+						id={this.state.friendIdToUpdate}
```

Comment:
I would just pass the whole state object in here using the spread operator {...this.state}, saves writing out each item